index.js

```diff
@@ -0,0 +1,41 @@
+const bodyParser = require('body-parser')
+const express = require('express')
+const teams = require('./teams.json')
+
+const app = express()
+
+app.get('/teams/', (request, response) => {
+    response.send(teams)
+})
+app.get('/teams/:input', (request, response) => {
+    const matchingTeams = teams.filter((team) => {
+        return team.id === parseInt(request.params.input) || team.abbreviation.toUpperCase() === request.params.input.toUpperCase()
+
+    })
+
+
+    if (matchingTeams.length) {
+        response.send(matchingTeams)
+    } else {
+        response.sendStatus(404)
+    }
+})
+
+app.use(bodyParser.json())
+
+app.post('/teams', (request, response) => {
+    const { id, location, mascot, abbreviation, conference, division } = request.body
+    if (!id || !location || !mascot || !abbreviation || !conference || !division) {
+
+        response.status(404).send('please provide information for all fields')
```

Comment:
I think you meant `400 Bad Request` here and not `404 Not Found`. Also I'd list the fields for the user, it's nicer that way.